lib/web_ui/dev/test_runner.dart

```diff
@@ -32,6 +32,13 @@ class TestsCommand extends Command<bool> {
         help: 'Pauses the browser before running a test, giving you an '
             'opportunity to add breakpoints or inspect loaded code before '
             'running the code.',
+      )
+      ..addFlag(
+        'update-screenshot-goldens',
+        defaultsTo: false,
+        help: 'When running screenshot tests writes them to the file system into '
+            '.dart_tool/goldens. Use this option to bulk-update all screenshots, '
+            'for example, when a new browser version affects pixels.',
```

Comment:
Does --update-screenshot-goldens work when using --target/-t? If so, can I use --update-screenshot-goldens when I need to update the goldens from a single target?

Reply:
Yep.

Reply:
Nice